src/bijectors/named_bijector.jl

```diff
@@ -25,21 +23,25 @@ julia> (a = 2 * x.a, b = exp(x.b), c = x.c)
 (a = 2.0, b = 1.0, c = 42.0)
 ```
 """
-struct NamedBijector{names, Bs<:NamedTuple{names}} <: AbstractNamedBijector
+struct NamedTransform{names, Bs<:NamedTuple{names}} <: AbstractNamedTransform
     bs::Bs
 end
 
 # fields contain nested numerical parameters
-function Functors.functor(::Type{<:NamedBijector{names}}, x) where names
+function Functors.functor(::Type{<:NamedTransform{names}}, x) where names
     function reconstruct_namedbijector(xs)
-        return NamedBijector{names,typeof(xs.bs)}(xs.bs)
+        return NamedTransform{names,typeof(xs.bs)}(xs.bs)
     end
     return (bs = x.bs,), reconstruct_namedbijector
 end
 
-names_to_bijectors(b::NamedBijector) = b.bs
+# TODO: Use recursion instead of `@generated`?
```

Comment:
Yeah, not a big fan of generated functions if they are not needed. Isn't it sufficient to use `map` here? I think usually that works quite well with `NamedTuple`.